source/projects/manage-relational-connection.txt

```diff
@@ -28,40 +28,44 @@ the :guilabel:`Manage` button.
 
 #. Click the check box next to the schema or schemas you are migrating. 
 
-#. To expand the tables in a schema, click ``>``.
+#. To expand the tables in a schema, click the :icon-fa5:`chevron-right`
+   button.
 
    Select the tables you want to add or remove from your migration.
 
 #. Click :guilabel:`Save`. 
 
    Changes from your relational database are reflected in the main 
-   Relational Migrator diagram and included in future migration jobs.
+   Relational Migrator diagram and included in future sync jobs.
 
 Refresh Schema Changes
 ----------------------
 
-If your relational database table structure changes, use the :guilabel:`Refresh Schema` button to let Relational Migrator know. 
+If your relational database table structure changes, use the 
+:guilabel:`Refresh Schema` button to let Relational Migrator know. 
 
-1. From the :guilabel:`Mapping` screen, click the :guilabel:`Manage` button on the 
-   :guilabel:`Schema model` pane.
+1. From the :guilabel:`Mapping` screen, click the :guilabel:`Manage` 
+   button on the :guilabel:`Schema model` pane.
 
-#. Click the :guilabel:`Refresh schema` button on the Manage relational model modal.
+#. Click the :guilabel:`Refresh schema` button on the Manage 
+   relational model modal.
 
-#. Select either :ref:`Connect DB <connect-db>` or :ref:`Import DDL File  <import-ddl-file>`.
+#. Select either :ref:`Connect to database <connect-to-db>` or 
+   :ref:`Import DDL File  <import-ddl-file>`.
 
-.. _connect-db:
+.. _connect-to-db:
 
-Connect DB 
-~~~~~~~~~~~
+Connect to database
```

Comment:
[suggestion]

I think we want to capitalize "Database" here.

Reply:
Unfortunately this is deliberate, the UI is string literal "Connect to database", if this changes I will update:

Reply:
<img width="587" alt="Screenshot 2023-08-11 at 3 34 22 PM" src="https://github.com/mongodb/docs-relational-migrator/assets/85948430/ff3afa58-e7fd-4ab8-a65c-2c183f6b44e7">